astore/client/auth/auth.go

```diff
@@ -103,5 +113,42 @@ func (c *Client) Login(username, domain string, o LoginOptions) (string, error)
 		return "", fmt.Errorf("could not decrypt returned token")
 	}
 
+	publicKey, _, _, _, err := ssh.ParseAuthorizedKey(tres.Capublickey)
```

Comment:
Side note, unrelated to your PR: loop from 88 to 103 should probably use the Retry library (did not have it when this code was written).

Reply:
I'll implement this in another PR, but yeah good point especially since the browser lib is pretty fickle